### ISSUE TITLE
Linux Process CPU Usage is not giving the real consumption

### ISSUE THREAD

**Issue by yusufozturk** [struCoder/pidusage#12]
First of all, I didn't check the output of this app. My assumptions are based on what I read from the code. If it's not correct, sorry for it.

For Linux, calculation here is based on processor usage of the process in the single logical core. So if one process utilize the entire core, you will always see 100% as CPU usage. But this is not the total CPU usage of the process, only the CPU usage in that core. Based on this calculation, you might see many processes with 100% cpu usage. And that doesn't sound right.

So basically, CPU usage should be divided by total number of logical cores to give real usage.

What do you think? Or do you think per core usage is better?
